modules/services/sharing/src/main/kotlin/au/com/shiftyjelly/pocketcasts/sharing/SharingAnalytics.kt

```diff
@@ -31,6 +31,7 @@ internal class SharingAnalytics(
         is SharingRequest.Data.ClipLink -> "clip_link"
         is SharingRequest.Data.ClipAudio -> "clip_audio"
         is SharingRequest.Data.ClipVideo -> "clip_video"
+        is SharingRequest.Data.WebLink -> "web_link"
```

Comment:
Doing it like this will log the `podcast_shared` event. If think it should use a different event.

Reply:
Passed separate (placeholder) analytics event and properties in a048e8a